window_installer/data/product_template.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="product_product_7_product_template" model="product.template" context="{'create_product_product': False}" forcecreate="1">
+    <field name="name">Travelling costs</field>
+    <field name="categ_id" ref="product.product_category_all"/>
+    <field name="uom_id" ref="uom.product_uom_km"/>
+    <field name="purchase_method">purchase</field>
+    <field name="uom_po_id" ref="uom.product_uom_km"/>
+    <field name="invoice_policy">order</field>
+    <field name="property_stock_inventory" ref="stock_location_14"/>
+    <field name="image_1920" type="base64" file="window_installer/static/src/binary/product_template/13-image_1920"/>
+    <field name="service_type">manual</field>
+    <field name="property_stock_production" ref="stock_location_15"/>
+    <field name="route_ids" eval="[(6, 0, [ref('purchase_stock.route_warehouse0_buy')])]"/>
+  </record>
```

Comment:
```suggestion
```
As we have kept this record in the product_product file so no need to define it here